hunts/chart_utils.py

```diff
@@ -0,0 +1,41 @@
+from django.utils import timezone
+
+from .models import Hunt
+from puzzles.models import Puzzle
+
+
+def can_use_chart(hunt):
+    if not hunt.start_time or timezone.now() <= hunt.start_time:
+        return False
+    return True
+
+
+def get_chart_data(hunt):
+    labels = ["Start"]
+    times = [hunt.start_time.isoformat()]
+    counts = [0]
+    is_meta = [False]
+
+    current_time = timezone.now()
+    if hunt.end_time:
+        current_time = min(current_time, hunt.end_time)
```

Comment:
Nit: Naming this `current_time` is confusing to read.

This corresponds to the chart end time right? Can we rename to `chart_end_time`?